GrandCentralBoard/Widgets/Harvest/View/CircleChartViewModel.swift

```diff
@@ -18,3 +18,49 @@ struct CircleChartViewModel {
     let startAngle: Double
     let items: [CircleChartItem]
 }
+
+extension CircleChartViewModel {
+
+    static func chartItemFromBillingStats(stats: DailyBillingStats) -> CircleChartViewModel {
```

Comment:
Can't `chartItemFromBillingStats` simply forward to `chartItemFromMultipleBillingStats`?

```swift
static func chartViewModelFromBillingStats(stats: DailyBillingStats) -> CircleChartViewModel {
    return chartViewModelFromMultipleBillingStats([stats])
}
```